wp-content/mu-plugins/https.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Plugin Name: HTTPS
  * Description: Fixes and improves HTTPS support.
- * Version:     1.0.0
+ * Version:     2.0.0
```

Comment:
Well, I don't think that this is a change for an increment of the major version. It's more like a minor version or a patch version. But definitely not a major version.